code/modules/projectiles/guns/smgs.dm

```diff
@@ -323,55 +377,83 @@
 	flags_equip_slot = SLOT_WAIST
 	fire_sound = 'sound/weapons/gun_uzi.ogg'
 	current_mag = /obj/item/ammo_magazine/smg/uzi
-	flags_gun_features = GUN_CAN_POINTBLANK|GUN_ANTIQUE
+	flags_gun_features = GUN_ANTIQUE|GUN_CAN_POINTBLANK|GUN_HAS_FULL_AUTO|GUN_FULL_AUTO_ON|GUN_FULL_AUTO_ONLY
```

Comment:
no more full auto guns are to be added until full auto code is improved

Reply:
vbtyuh